examples/plot_group_lasso.py

```diff
@@ -145,11 +138,13 @@ def process_meg(raw_name):
 # it must be set as a positive number between 0 and 1. With alpha = 1, all
 # the sources are 0.
 
-stcs, log = compute_group_inverse(gains, M, group_info,
-                                  method="grouplasso",
-                                  depth=0.9, alpha=0.6, return_stc=True,
-                                  n_jobs=4)
+inv_op = InverseOperator(fwds, noise_covs, src_ref, ch_type="grad",
+                         depth=0.9)
+inv_op.compute_group_model()
```

Comment:
I would avoid an object and try to do something like:

```
fwds = prepare_forwards(fwds, src_ref)
stcs = computer_group_inverse(fwds, evokeds, noise_covs, depth=0.9, method="grouplasso")
```

I would take the tmin, tmax from the evokeds imposing that they have the same times attribute.
I would call time_independent -> spatiotemporal= True | False

if True the solver works with both space and time.

Reply:
The problem with this is the overhead of aligning the forwards (first aligning and then checking all vertices that were thrown away by the BEM for each subject and then compute the intersection of the vertices). Recomputing an stc with a different hyperparam or solver would require doing this all over again ... 
Ok tmin / tmax; and  spatiotemporal, I like it